go/signedexchange/README.md

```diff
@@ -0,0 +1,35 @@
+# gen-signedexchange
+
+This directory holds a fork of the [gen-signedexchange tool](https://github.com/WICG/webpackage/tree/master/go/signedexchange) that best matches Chrome implementation of now.
```

Comment:
"best matches Chrome implementation of now" -> "best matches the current Chrome implementation"?